core/src/templates/components/card/card.twig

```diff
@@ -3,78 +3,94 @@
- * - card_super_headline: Text heading
+ * - card_super_headline: Text super headline. Often used for metadata such as category.
  * - card_headline: Text heading
  * - card_body: An open variable for anything. Generally this would be for
  *     WYSIWYG editor contents.
- * - card_cta_attributes: Any html attributes on the cta link that need to be
+ * - card_cta_attributes: Any html attributes on the link that need to be
  *     added. For example: _target.
  * - card_cta_label: The clickable text for the CTA link
  * - card_button_label: The text label of the button component.
  */
 #}
-<div{{ attributes }} class="su-card {{ modifier_class }}">
 
-  {# Card Image #}
-  {% if card_image is not empty -%}
-    {{ card_image }}
+{# For variant where whole card is a link, change the outer wrapper from <div> to <a> #}
+{% if "su-card--link" in modifier_class %}
+  {% set allow_links = false %}
+{% else %}
+  {% set allow_links = true %}
+{%- endif -%}
+
+<div {{ attributes }} class="su-card {{ modifier_class }}">
+  {%- if allow_links == false %}
+    <a href="{{ card_link }}" {{ card_cta_attributes }}>
   {%- endif -%}
 
-  <div class="su-card__contents">
+    {# Card Image #}
+    {% if card_image is not empty -%}
+      {{ card_image }}
+    {%- endif -%}
 
-  {# Card Super Headline #}
-  {%- block block_card_super_headline %}
-    {% if card_super_headline is not empty %}
-      <h4>{{ card_super_headline }}</h4>
-    {% endif -%}
-  {% endblock -%}
+    <div class="su-card__contents">
 
-  {# Card Headline #}
-  {%- block block_card_headline  %}
-    {%- if card_headline is not empty %}
-      <h3>{{ card_headline }}</h3>
-    {% endif -%}
-  {% endblock -%}
+    {# Card Super Headline #}
+    {%- block block_card_super_headline %}
+      {% if card_super_headline is not empty %}
+        <span>{{ card_super_headline }}</span>
+      {% endif -%}
+    {% endblock -%}
 
-  {# An open field for body content #}
-  {{ card_body }}
+    {# Card Headline #}
+    {%- block block_card_headline  %}
+      {%- if card_headline is not empty %}
+        <h3>{{ card_headline }}</h3>
+      {% endif -%}
+    {% endblock -%}
 
-  {# Card CTA Link #}
-  {%- if card_cta_label is not empty %}
-    {%- include "@decanter/components/link/link.twig" with
-      {
-      "link_href": card_link,
-      "link_content": card_cta_label,
-      "attributes": card_cta_attributes,
-      "modifier_class": "su-card--link su-link--action",
-      }
-      only
-    -%}
-  {% endif -%}
+    {# An open field for body content #}
+    {{ card_body }}
 
-  {# Big ol button. Adds a BEM class for targeting. #}
-  {%- if card_button_label is not empty %}
-    <div class="su-card__button">
-    {%- include "@decanter/components/link/link.twig" with
-      {
-      "link_href": card_link,
-      "link_content": card_button_label,
-      "modifier_class": "su-button",
-      }
-      only
-    -%}
-    </div>
-  {% endif -%}
+    {# Card CTA Link #}
```

Comment:
Would we still want to visually render the link or button even when the whole card is wrapped?

Reply:
Had a discussion with @kerri-augenstein yesterday and we were ok with not rendering the a button/link when whole card is wrapped in a link - but it's nice to get another confirmation from her.